src/main/java/com/github/jmchilton/blend4j/galaxy/WorkflowsClientImpl.java

```diff
@@ -42,13 +44,25 @@ public WorkflowOutputs runWorkflow(final WorkflowInputs workflowInputs) {
     return runWorkflowResponse(workflowInputs).getEntity(WorkflowOutputs.class);
   }
 
-  public ClientResponse importWorkflowResponse(final String json) {
-    final String payload = String.format("{\"workflow\": %s}", json);
+  public ClientResponse runWorkflowInvocationResponse(WorkflowInvocationInputs workflowInvocationInputs) {
+    return create(getWebResource().path(workflowInvocationInputs.getWorkflowId()).path("invocations"),workflowInvocationInputs);
+  }
+
+  public WorkflowInvocationOutputs invokeWorkflow(final WorkflowInvocationInputs workflowInvocationInputs) {
```

Comment:
Thanks so much @JeffreyThiessen. Overall this looks great.

One comment I have for this class, could you add a `@Deprecated` annotation to the `runWorkflow()` and `runWorkflowResponse()` methods?

Reply:
Good catch. Added.